### PR TITLE
Added smitshrestha101 to virtual intern list

### PR DESCRIPTION
**Pull request by smitshrestha101**
### Description
Adding myself to the virtual intern list

### Raw.Githack preview link

https://raw.githack.com/smitshrestha101/smitshrestha101.github.io/add-intern/#!pages/vi/vi-team.md

### CODE REVIEW

**Comment by ChiragAwale** [comment]
@lmmrssa  @dogi @rrijal53

**Comment by lmmrssa** [comment]
@smitshrestha101 you should try to clean this PR.

File: pages/manual/myplanet/overview.md [hunk 1]
```diff
@@ -1 +1 @@
-**We are in the process of updating this page!**
+**We are in the process of updating this page!**
```
Comment (dogi) [inline pages/manual/myplanet/overview.md]: hmmm

**Comment by smitshrestha101** [comment]
@dogi @lmmrssa @rrijal53 @ChiragAwale (https://github.com/open-learning-exchange/open-learning-exchange.github.io/pull/2732). I have created this pull request that has fixed the above problem. It was a problem that kept showing a change in overview.md even when i did not make it. Had to run git rm --cached -r . to fix the problem.

**Pull request closed without merging**